README.md

```diff
@@ -53,13 +53,20 @@ A sub-section of all our collections have been identified and can be queued up
 
   `bundle exec rake pulfalight:aspace:index_test_eads`
 
+#### Index a specific EAD
+
+In a rails console run the index job with a specific EAD, e.g.:
+```
+AspaceIndexJob.perform_later(resource_descriptions_uri: "repositories/4/resource_descriptions/2203", repository_id: "univarchives")
+```
+
+Note the uri must use a collection-level resource id (you may need to look this up in space), and have the correct corresponding repository_id (check against config/repositories.yml)
```

Comment:
Should this read "(you may need to look this up in Aspace)"?

Reply:
Yep, thank you. Fixed!